main.go

```diff
@@ -66,10 +70,15 @@ BE ON THE LOOKOUT !
 		for {
 
 			var input string
-			if _, err := fmt.Scanf("%s\n", &input); err != nil {
 
+			char, key, err := keyboard.GetSingleKey()
+			if err != nil {
 				log.Println(err)
 			}
+			defer fmt.Printf("snap: %q\r\n", char)
+			if key == keyboard.KeyEsc {
```

Comment:
If I press Esc I see:

```
snap: '\x00'
snap: 'd'
snap: 'a'
snap: 's'
snap: 'a'
snap: 'd'
```

from the defered print, and then the cards keep being dealt, but I cannot snap any more. This seems to be a bug. Either pressing `Esc` should stop the whole game, or it should do nothing.